Modules/_interpqueuesmodule.c

```diff
@@ -1603,7 +1603,7 @@ queuesmod_put(PyObject *self, PyObject *args, PyObject *kwds)
     PyObject *obj;
     int unboundarg = -1;
     int fallbackarg = -1;
-    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O&O|ii$p:put", kwlist,
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O&O|ii:put", kwlist,
```

Comment:
I think that this is the correct solution:
```suggestion
    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O&O|i$p:put", kwlist,
```
Please add tests with the `fallback` argument equal to `None`, `-1`, `2**1000` -- there should be different behavior for patched and unpatched code. Also, `fallback` is now keyword-only. And passing unsupported keyword argument could crash the old code.

Reply:
I went with 'i' because that's what `.create()` uses, but I'm now not sure which is more correct. @ericsnowcurrently?

Reply:
#134440 removed `fmt` ("i") and added keyword-only boolean `fallback` ("p"). We usually make booleans keyword-only.

`fallback` has only three state -- _PyXIDATA_XIDATA_ONLY (0), _PyXIDATA_FULL_FALLBACK (1), and default negative value used to distinguish "no argument" from passed value. Other positive values are errors.

If it was also added in other functions, it most likely should be keyword-only boolean too.

Reply:
I've done as you suggest. Note the changes I had to make to `.create()`, though.

It would be good to get this in to 3.14rc2 (now this Thu). Feel free to take over the branch.